src/core/css/SearchResult.scss

```diff
@@ -23,7 +23,15 @@
 }
 
 .SearchResult-main {
+  display: flex;
+  flex-direction: row;
   flex-grow: 1;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.SearchResult-text {
+  width: 90%;
```

Comment:
I used % because what I've observed is that it makes responsiveness better compared to when px values are used. I've faced issues earlier in my projects with resizing when using pixels but that becomes fine with the use of %. So following that notion I used %.